site/build-and-publish.sh

```diff
@@ -22,11 +22,11 @@ export BRANCH COMMIT OUTPUT
 bundle install --path=.bundle
 bundle exec jekyll build --verbose
 
-gsutil -m cp -z html,css -a public-read -R _site "gs://docs.weave.works/${OUTPUT}"
+gsutil -m rsync -d _site "gs://docs.weave.works/${OUTPUT}"
```

Comment:
The `-z` and `-a` flags are not supported, but we shouldn't have too big files to compress them anyway and now a default permission set on the bucket for all object to be publicly readable, so `-a` is not needed also.